scripts/firedrake-install

```diff
@@ -298,6 +300,11 @@ def run_pip(args):
 
 
 def run_pip_install(args):
+    if args.log:
```

Comment:
This is bound to the wrong `args` here, as the function argument is also called `args`.